<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QInputVector3Dialog</class>
 <widget class="QDialog" name="QInputVector3Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>47</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>227</red>
        <green>227</green>
        <blue>227</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>227</red>
        <green>227</green>
        <blue>227</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>227</red>
        <green>227</green>
        <blue>227</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowOpacity">
   <double>0.900000000000000</double>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QDoubleSpinBox" name="zval_spinbox_">
     <property name="minimum">
      <double>-99999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>99999999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="yval_spinbox_">
     <property name="minimum">
      <double>-99999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>999999999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="xval_spinbox_">
     <property name="minimum">
      <double>-99999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>99999999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>x           y           z</string>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>label</zorder>
  <zorder>xval_spinbox_</zorder>
  <zorder>yval_spinbox_</zorder>
  <zorder>zval_spinbox_</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
